bundles/specmate-persistency-cdo/src/com/specmate/persistency/cdo/internal/CDOPersistencyService.java

```diff
@@ -281,19 +284,39 @@ public IView openView() {
 		return view;
 	}
 
-	private boolean readConfig(Config config) {
-		jdbcConnection = config.cdoJDBCConnection();
-		repository = config.cdoRepositoryName();
-		resourceName = config.cdoResourceName();
-		userResourceName = config.cdoUserResourceName();
-		if (StringUtils.isEmpty(repository) || StringUtils.isEmpty(resourceName)
-				|| StringUtils.isEmpty(userResourceName)) {
+	private boolean readConfig(Config defaultConfig) {

```

Comment:
the config that is provided here is not the default config but the config from the configuration admin. In fact, the configuration admin calls the activate method with the saved config.